dashboards/ds-solarwinds-webapi/dashboard.json

```diff
@@ -0,0 +1,418 @@
+                          },
+                          "SysName": {
+                            "displayName": "Node"
+                          },
+                          "Vendor": {
+                            "displayName": "Vendor"
+                          },
+                          "Description": {
+                            "displayName": "Description"
+                          },
+                          "Location": {
+                            "displayName": "Location"
+                          },
+                          "CPUCount": {
+                            "displayName": "# CPU"
+                          },
+                          "IPAddress": {
+                            "displayName": "IP Address"
+                          },
+                          "AvgResponseTime": {
+                            "displayName": "Response Time",
+                            "template": "{{value}} ms"
+                          },
+                          "LastBoot": {
+                            "displayName": "Last Boot",
+                            "template": "{{timeago(value)}}"
+                          },
+                          "StatusDescription": {
+                            "displayName": "Summary",
+                            "template": "{{value.substr(0,100)}}"
+                          }
+                        },
+                        "customColumns": {},
+                        "columns": [
+                          "Status",
+                          "SysName",
+                          "Vendor",
+                          "Description",
+                          "Location",
+                          "CPUCount",
+                          "IPAddress",
+                          "AvgResponseTime",
+                          "LastBoot",
+                          "StatusDescription"
+                        ],
+                        "autohide": false,
+                        "rowLink": "http://int-solarwinds.int.squaredup.com:8787/Orion/NetPerfMon/NodeDetails.aspx?NetObject=N:{{NodeID}}"
+                      }
+                    },
+                    "description": "",
+                    "title": "All Nodes"
+                  }
+                ]
+              }
+            ],
+            "width": 75.146
+          }
+        ]
+      }
+    ]
+  },
+  "derivedFrom": "00000000-0000-0000-0000-000000000000",
+  "draft": true,
+  "openAccess": {
```

Comment:
Also not sure if we publish dashboards with OA enabled by default.

Reply:
This should not be enabled by default. Providing unauthenticated access to SolarWinds data would add our nails to their coffin.

Reply:
Agree. That has already been set to false on a recent commit. Thanks.